sdk/python/packages/flet-web/src/flet_web/fastapi/app.py

```diff
@@ -42,16 +42,16 @@ def app(
     Mount all Flet FastAPI handlers in one call.
 
     Parameters:
-    * `session_handler` (function or coroutine) - application entry point - a method
+    * `main` (function or coroutine) - application entry point - a method
        called for newly connected user. Handler must have 1 parameter: `page` - `Page`
        instance.
+    * `before_main` - a function that is called after Page was created, but before
+       calling `main`.
     * `assets_dir` (str, optional) - an absolute path to app's assets directory.
     * `app_name` (str, optional) - PWA application name.
     * `app_short_name` (str, optional) - PWA application short name.
     * `app_description` (str, optional) - PWA application description.
     * `web_renderer` (WebRenderer) - web renderer defaulting to `WebRenderer.AUTO`.
-    * `use_color_emoji` (bool) - whether to load a font with color emoji.
-       Default is `False`.
```

Comment:
May I please know how color-emojis are now handled, for this prop to no longer be needed?

Reply:
`useColorEmoji` has gone from Flutter! :) I've searched the entire Flutter repo of 3.29.3 branch and nothing.
There are two renderers now: CanvasKit and Skwasm and from what I see they are just loading emoji font (Noto) with "no questions asked". You can't disable that. But! There is a bonus: I found a "hidden" config flag to redirect fallback fonts from CDN to any URL.